### PR TITLE
Issues #464 and #466 (admin panel navigation)

### PR DESCRIPTION
**Pull request by Ziachu**
**Story / Bug id:** #464
**Story / Bud id:** #465 
**Story / Bud id:** #466 
**Reference person:** @Ziachu
**Description:**
- Base admin panel navigation created. 
- Templates (offers/archived.html, offers/offers_list.html) now does show admin navigation panel
- Admin have direct link to admin panel (when clicks "Witaj, ...")
- Admin after login sees main homepage

**New imports / dependencies:**

**Unit Tests:**

None

**What tests do I need to run to validate this change:**

Description of testing workflow and potential affected places.

### CODE REVIEW

**Comment by radoslawroszkowiak** [comment]
Please sync it with current master so that there were no conflicts

File: apps/volontulo/templates/common/header.html [hunk 1]
```diff
@@ -25,6 +25,11 @@ <h1 class="navbar-header">
                             <li>
                                 <a href="{% url 'logged_user_profile' %}">Moje konto</a>
                             </li>
+                            {% if user.is_staff %}
+                            <li>
+				                <a href="{% url 'admin_panel' %}">Panel administracyjny</a>
```
Comment (radoslawroszkowiak) [inline apps/volontulo/templates/common/header.html]: strange indentation

File: apps/volontulo/views/admin_panel.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+"""
+
+from django.shortcuts import render
+
+def main_panel(request):
+
+	return render(
+		request,
+		'admin/list_offers.html'
+	)
```
Comment (tomaszwozniak) [inline apps/volontulo/views/admin_panel.py]: missing new line at the end file

File: apps/volontulo/views/admin_panel.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+"""
+.. module:: admin_panel
+"""
+
+from django.shortcuts import render
+
+def main_panel(request):
+
```
Comment (tomaszwozniak) [inline apps/volontulo/views/admin_panel.py:9]: unnecessary blank line

**Comment by tomaszwozniak** [comment]
Minor comments

File: apps/volontulo/templates/common/header.html (hunk 1 above)
Comment (filipgorczynski) [inline apps/volontulo/templates/common/header.html]: You can use ?ts=4 in url to treat tab as 4 spaces... but, here we have only 3 spaces so should be fixed.

**Comment by tomaszwozniak** [comment]
0.87s$ pep8 --exclude='apps/volontulo/migrations/*,node_modules,.ropeproject' .
./apps/volontulo/views/admin_panel.py:6:1: E302 expected 2 blank lines, found 1
./apps/volontulo/views/admin_panel.py:8:1: W191 indentation contains tabs
./apps/volontulo/views/admin_panel.py:9:1: W191 indentation contains tabs
./apps/volontulo/views/admin_panel.py:10:1: W191 indentation contains tabs
./apps/volontulo/views/admin_panel.py:11:1: W191 indentation contains tabs

**Comment by tomaszwozniak** [comment]
+------------------+------------+
|message id        |occurrences |
+==================+============+
|locally-disabled  |119         |
+------------------+------------+
|missing-docstring |1           |
+------------------+------------+

**Comment by tomaszwozniak** [comment]
looks good

**Pull request merged**